package.json

```diff
@@ -1,119 +1,27 @@
-    "density-clustering": "^1.3.0",
-    "eventemitter3": "^4.0.7",
-    "exifr": "^7.1.3",
-    "ffmpeg-wasm": "file:./thirdparty/ffmpeg-wasm",
-    "file-type": "^16.5.4",
-    "formik": "^2.1.5",
-    "get-user-locale": "^2.1.3",
-    "hdbscan": "0.0.1-alpha.5",
-    "heic-convert": "^1.2.4",
-    "http-proxy-middleware": "^1.0.5",
-    "i18next": "^22.4.11",
-    "i18next-http-backend": "^2.1.1",
-    "idb": "^7.0.0",
-    "is-electron": "^2.2.0",
-    "jszip": "3.8.0",
-    "libsodium-wrappers": "^0.7.8",
-    "localforage": "^1.9.0",
-    "ml-matrix": "^6.8.2",
-    "next": "^13.1.2",
-    "next-transpile-modules": "^10.0.0",
-    "otpauth": "^9.0.2",
-    "p-queue": "^7.1.0",
-    "photoswipe": "file:./thirdparty/photoswipe",
-    "piexifjs": "^1.0.6",
-    "react": "^18.2.0",
-    "react-bootstrap": "^1.3.0",
-    "react-datepicker": "^4.3.0",
-    "react-dom": "^18.2.0",
-    "react-dropzone": "^11.2.4",
-    "react-i18next": "^12.2.0",
-    "react-otp-input": "^2.3.1",
-    "react-select": "^4.3.1",
-    "react-simple-code-editor": "^0.11.0",
-    "react-top-loading-bar": "^2.0.1",
-    "react-virtualized-auto-sizer": "^1.0.2",
-    "react-window": "^1.8.6",
-    "sanitize-filename": "^1.6.3",
-    "similarity-transformation": "^0.0.1",
-    "transformation-matrix": "^2.10.0",
-    "vscode-uri": "^3.0.7",
-    "workbox-precaching": "^6.1.5",
-    "workbox-recipes": "^6.1.5",
-    "workbox-routing": "^6.1.5",
-    "workbox-strategies": "^6.1.5",
-    "workbox-window": "^6.1.5",
-    "xml-js": "^1.6.11",
-    "yup": "^0.29.3",
-    "zxcvbn": "^4.4.2"
-  },
-  "devDependencies": {
-    "@next/bundle-analyzer": "^13.1.6",
-    "@types/bs58": "^4.0.1",
-    "@types/debounce-promise": "^3.1.3",
-    "@types/libsodium-wrappers": "^0.7.8",
-    "@types/node": "^14.6.4",
-    "@types/photoswipe": "^4.1.1",
-    "@types/react": "^16.9.49",
-    "@types/react-collapse": "^5.0.1",
-    "@types/react-datepicker": "^4.1.7",
-    "@types/react-select": "^4.0.15",
-    "@types/react-window": "^1.8.2",
-    "@types/react-window-infinite-loader": "^1.0.3",
-    "@types/wicg-file-system-access": "^2020.9.5",
-    "@types/yup": "^0.29.7",
-    "@types/zxcvbn": "^4.4.1",
-    "@typescript-eslint/eslint-plugin": "^5.43.0",
-    "eslint": "^8.28.0",
-    "eslint-config-next": "^13.0.6",
-    "eslint-config-prettier": "^8.5.0",
-    "husky": "^7.0.1",
-    "lint-staged": "^11.1.2",
-    "prettier": "2.3.2",
-    "typescript": "^4.1.3"
-  },
-  "standard": {
-    "parser": "babel-eslint"
-  }
-}
+    "name": "bada-frame",
+    "version": "1.0.0",
+    "private": true,
+    "scripts": {
+        "build": "turbo run build && turbo run export",
+        "dev": "turbo run dev --parallel",
+        "lint": "turbo run lint",
+        "albums": "next dev -p 3002",
+        "auth": "next dev -p 3004",
+        "export": "next export",
+        "build-analyze": "ANALYZE=true next build",
+        "start": "next start",
+        "prepare": "husky install"
```

Comment:
I assume it would need to have `husky` installed to be at the root level. for this to work?

Also we currently lint the app on `pre-commit ` hook. Can you check how that will work, such that the app-specific eslint-config is used

Reply:
I've fixed linting and made sure they use the correct configs, so it should work. The husky pre-commit command simply runs the lint-staged command, which I assume uses the same logic as the normal lint command.

Reply:
but shouldn't it be in the `package.json` devDependency?

Also, how is lint-staged running that is also not added as a dependency?

also the `.lintstagedrc.js`  runs `next lint` and `yarn prettier` commands how are they present without them being added as dependency?